linux_os/guide/system/accounts/accounts-physical/disable_ctrlaltdel_burstaction/ignition/shared.yml

```diff
@@ -0,0 +1,15 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    storage:
+      files:
+      - contents:
+          source: data:,%23%20%20This%20file%20is%20part%20of%20systemd.%0A%23%0A%23%20%20systemd%20is%20free%20software%3B%20you%20can%20redistribute%20it%20and/or%20modify%20it%0A%23%20%20under%20the%20terms%20of%20the%20GNU%20Lesser%20General%20Public%20License%20as%20published%20by%0A%23%20%20the%20Free%20Software%20Foundation%3B%20either%20version%202.1%20of%20the%20License%2C%20or%0A%23%20%20%28at%20your%20option%29%20any%20later%20version.%0A%23%0A%23%20Entries%20in%20this%20file%20show%20the%20compile%20time%20defaults.%0A%23%20You%20can%20change%20settings%20by%20editing%20this%20file.%0A%23%20Defaults%20can%20be%20restored%20by%20simply%20deleting%20this%20file.%0A%23%0A%23%20See%20systemd-system.conf%285%29%20for%20details.%0A%0A%5BManager%5D%0A%23LogLevel%3Dinfo%0A%23LogTarget%3Djournal-or-kmsg%0A%23LogColor%3Dyes%0A%23LogLocation%3Dno%0A%23DumpCore%3Dyes%0A%23ShowStatus%3Dyes%0A%23CrashChangeVT%3Dno%0A%23CrashShell%3Dno%0A%23CrashReboot%3Dno%0ACtrlAltDelBurstAction%3Dnone%0A%23CPUAffinity%3D1%202%0A%23JoinControllers%3Dcpu%2Ccpuacct%20net_cls%2Cnet_prio%0A%23NUMAPolicy%3Ddefault%0A%23NUMAMask%3D%0A%23RuntimeWatchdogSec%3D0%0A%23ShutdownWatchdogSec%3D10min%0A%23CapabilityBoundingSet%3D%0A%23NoNewPrivileges%3Dno%0A%23SystemCallArchitectures%3D%0A%23TimerSlackNSec%3D%0A%23DefaultTimerAccuracySec%3D1min%0A%23DefaultStandardOutput%3Djournal%0A%23DefaultStandardError%3Dinherit%0A%23DefaultTimeoutStartSec%3D90s%0A%23DefaultTimeoutStopSec%3D90s%0A%23DefaultRestartSec%3D100ms%0A%23DefaultStartLimitIntervalSec%3D10s%0A%23DefaultStartLimitBurst%3D5%0A%23DefaultEnvironment%3D%0A%23DefaultCPUAccounting%3Dno%0A%23DefaultIOAccounting%3Dno%0A%23DefaultIPAccounting%3Dno%0A%23DefaultBlockIOAccounting%3Dno%0A%23DefaultMemoryAccounting%3Dyes%0A%23DefaultTasksAccounting%3Dyes%0A%23DefaultTasksMax%3D80%25%0A%23DefaultLimitCPU%3D%0A%23DefaultLimitFSIZE%3D%0A%23DefaultLimitDATA%3D%0A%23DefaultLimitSTACK%3D%0A%23DefaultLimitCORE%3D%0A%23DefaultLimitRSS%3D%0A%23DefaultLimitNOFILE%3D%0A%23DefaultLimitAS%3D%0A%23DefaultLimitNPROC%3D%0A%23DefaultLimitMEMLOCK%3D%0A%23DefaultLimitLOCKS%3D%0A%23DefaultLimitSIGPENDING%3D%0A%23DefaultLimitMSGQUEUE%3D%0A%23DefaultLimitNICE%3D%0A%23DefaultLimitRTPRIO%3D%0A%23DefaultLimitRTTIME%3D%0A%23IPAddressAllow%3D%0A%23IPAddressDeny%3D%0A%0A
+        filesystem: root
+        mode: 0644
+        path: /etc/systemd/system.conf
```

Comment:
Could ``/etc/systemd/system.conf.d/*.conf`` be used?

Reply:
(ref https://www.freedesktop.org/software/systemd/man/systemd-system.conf.html)  .... have not checked if this is on RHCOS though. Will look in the morning.

Reply:
@anixon-rh: confirmed, ``/etc/systemd/system.conf.d`` is present on CoreOS